test/integration/serverless/snapshots/amd64/log-csharp

```diff
@@ -1,12 +1,12 @@
+[log] {"message":{"message":"2022-03-09T15:40:37.296Z\t4c44fe51-c452-4951-9d2d-956d2edd5c87\tinfo\tXXX Log 2 XXX\n","lambda":{"arn":"arn:aws:lambda:eu-west-1:601427279990:function:integration-tests-extension-STAGE-log-csharp"}},"status":"info","timestamp":TIMESTAMP,"hostname":"arn:aws:lambda:eu-west-1:601427279990:function:integration-tests-extension-STAGE-log-csharp","service":"integration-tests-service","ddsource":"lambda","ddtags":"account_id:601427279990,account_id:601427279990,account_id:601427279990,architecture:x86_64,architecture:x86_64,architecture:x86_64,aws_account:601427279990,aws_account:601427279990,aws_account:601427279990,dd_extension_version:123,dd_extension_version:123,dd_extension_version:123,env:integration-tests-env,env:integration-tests-env,env:integration-tests-env,function_arn:arn:aws:lambda:eu-west-1:601427279990:function:integration-tests-extension-STAGE-log-csharp,function_arn:arn:aws:lambda:eu-west-1:601427279990:function:integration-tests-extension-STAGE-log-csharp,function_arn:arn:aws:lambda:eu-west-1:601427279990:function:integration-tests-extension-STAGE-log-csharp,functionname:integration-tests-extension-STAGE-log-csharp,functionname:integration-tests-extension-STAGE-log-csharp,functionname:integration-tests-extension-STAGE-log-csharp,memorysize:1024,memorysize:1024,memorysize:1024,region:eu-west-1,region:eu-west-1,region:eu-west-1,resource:integration-tests-extension-STAGE-log-csharp,resource:integration-tests-extension-STAGE-log-csharp,resource:integration-tests-extension-STAGE-log-csharp,runtime:dotnet6,runtime:dotnet6,runtime:dotnet6,service:integration-tests-service,service:integration-tests-service,service:integration-tests-service,taga:valuea,taga:valuea,taga:valuea,tagb:valueb,tagb:valueb,tagb:valueb,tagc:valuec,tagc:valuec,tagc:valuec,tagd:valued,tagd:valued,tagd:valued,version:integration-tests-version,version:integration-tests-version,version:integration-tests-version"}
```

Comment:
It looks like the logs aren't getting successfully normalized, which is why log-csharp and trace-csharp integration tests are failing. 

The log should appear as `XXX Log 0 XXX`, but it's 
`2022-03-09T15:40:36.789Z\t4c44fe51-c452-4951-9d2d-956d2edd5c87\tinfo\tXXX Log 0 XXX`. Maybe that extra timestamp is added with the dotnet6.0 support?

Reply:
We may need to add another line here: https://github.com/DataDog/datadog-agent/blob/b7be79b0bada1071ebe1da5ffd8addca732a3ce4/test/integration/serverless/run.sh#L235

for log and trace test output normalization